src/main/kotlin/org/javacs/kt/classpath/findClassPath.kt

```diff
@@ -19,11 +19,19 @@ import java.util.Comparator
 import java.util.concurrent.TimeUnit
 
 fun findClassPath(workspaceRoots: Collection<Path>): Set<Path> {
-    return workspaceRoots
+    return ensureStdLibInPaths(workspaceRoots
             .flatMap { projectFiles(it) }
             .flatMap { readProjectFile(it) }
             .toSet()
             .ifEmpty { backupClassPath() }
```

Comment:
We could remove this line as its functionality is already provided by `ensureStdLibinPaths`

Reply:
Good pick up 👍